app/dashboard/models.py

```diff
@@ -987,6 +988,7 @@ class BountyFulfillment(SuperModel):
     fulfillment_id = models.IntegerField(null=True, blank=True)
     fulfiller_hours_worked = models.DecimalField(null=True, blank=True, decimal_places=2, max_digits=50)
     fulfiller_github_url = models.CharField(max_length=255, blank=True, null=True)
+    funder_last_notified_on = models.DateTimeField(null=True, blank=True)
```

Comment:
Do we need two properties to track this? 🤔

Reply:
@pinkiebell I used one property in the batch job that runs and sends automatic emails. These are run separately from the fulfiller's notices, as these are Gitcoin notices. 

I used another property just for when the fulfiller has attempted to contact the funder. I think this was what was stated in the requirements!